src/plugins/filter.ts

```
@@ -584,26 +561,17 @@ export async function apply(ctx: Context, config: Config) {
-        }
+    ctx.setInterval(() => {
+        void Promise.allSettled(
+            Object.keys(groupInfos).map((guildId) =>
+                processSchedulerTickForGuild(ctx, config, guildId).catch((e) => {
+                    logger.error(
+                        `[next_reply] scheduler failed guild=${guildId}`,
+                        e
+                    )
+                })
+            )
+        )
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)